pkg/status/status_server.go

```diff
@@ -50,11 +50,11 @@ const (
 	patternHelp               = "/help"
 	patternOptions            = "/options"
 	patternVersion            = "/version"
-	patternBpfAdsMaps         = "/debug/config_dump/bpf/ads"
-	patternBpfWorkloadMaps    = "/debug/config_dump/bpf/workload"
+	patternBpfAdsMaps         = "/debug/config_dump/bpf/kernel-native"
+	patternBpfWorkloadMaps    = "/debug/config_dump/bpf/dual-engine"
 	configDumpPrefix          = "/debug/config_dump"
-	patternConfigDumpAds      = configDumpPrefix + "/ads"
-	patternConfigDumpWorkload = configDumpPrefix + "/workload"
+	patternConfigDumpAds      = configDumpPrefix + "/kernel-native"
+	patternConfigDumpWorkload = configDumpPrefix + "/dual-engine"
```

Comment:
Good fix, it would be better to add UT for this subcommand, then similar issue would not occur.

Reply:
Yeah sure. I will add UT for this command.